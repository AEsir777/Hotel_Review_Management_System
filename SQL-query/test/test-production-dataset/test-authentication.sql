/**** authentication ******/
/*** Test insert (mostly already tested when inserting into) ******/
CALL InsertIntoUserAuth('123e4567-e89b-12d3-a456-426614174000', 'test1@example.com', 'password');
CALL InsertIntoUserAuth('123e4567-e89b-12d3-a456-426614171234', 'test2@example.com', 'pwd');

/*** Test get user ***/
CALL GetUserByUid('123e4567-e89b-12d3-a456-426614174000');
CALL GetUserByUid('123e4567-e89b-12d3-a456-426614171234');
CALL GetUserByUid('w-zD8Ln3XZszM82AfVrspg');
CALL GetUserByUid('qvmL6mp-rpIwVguxb5HuvQ');
CALL GetUserByUid('eTvp_hYnsrI5-ow_sQ31_g');
CALL GetUserByUid('GG0mFsEXb-02_dzFPqRV1Q');
CALL GetUserByUid('not exist');
