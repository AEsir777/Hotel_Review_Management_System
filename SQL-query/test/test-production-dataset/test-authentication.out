/**** authentication ******/
/*** Test insert (mostly already tested when inserting into) ******/
// CALL InsertIntoUserAuth('123e4567-e89b-12d3-a456-426614174000', 'test1@example.com', 'password');
1 row(s) affected
// expected output: 1 row(s) affected

// CALL InsertIntoUserAuth('123e4567-e89b-12d3-a456-426614171234', 'test2@example.com', 'pwd');
1 row(s) affected
// expected output: 1 row(s) affected

/*** Test get user ***/
// CALL GetUserByUid('123e4567-e89b-12d3-a456-426614174000');
1 row(s) returned
123e4567-e89b-12d3-a456-426614174000	test1@example.com	password
//expected output: 123e4567-e89b-12d3-a456-426614174000	test1@example.com	password

// CALL GetUserByUid('123e4567-e89b-12d3-a456-426614171234');
1 row(s) returned
'123e4567-e89b-12d3-a456-426614171234', 'test2@example.com', 'pwd'
//expected output: '123e4567-e89b-12d3-a456-426614171234', 'test2@example.com', 'pwd'

// CALL GetUserByUid('"w-zD8Ln3XZszM82AfVrspg"');
1 row(s) returned
"w-zD8Ln3XZszM82AfVrspg"	jowttpke@gmail.com	LQdk4nrs
//expected output: "w-zD8Ln3XZszM82AfVrspg"	jowttpke@gmail.com	LQdk4nrs

// CALL GetUserByUid('"qvmL6mp-rpIwVguxb5HuvQ"');
1 row(s) returned
"qvmL6mp-rpIwVguxb5HuvQ"	oBALADF6@gmail.com	Pl9JtQEbpr
//expected output: "qvmL6mp-rpIwVguxb5HuvQ"	oBALADF6@gmail.com	Pl9JtQEbpr

// CALL GetUserByUid('"eTvp_hYnsrI5-ow_sQ31_g"');
1 row(s) returned
"eTvp_hYnsrI5-ow_sQ31_g"	dwlphaoy@gmail.com	01EpfXrW
//expected output: "eTvp_hYnsrI5-ow_sQ31_g"	dwlphaoy@gmail.com	01EpfXrW

// CALL GetUserByUid('"GG0mFsEXb-02_dzFPqRV1Q"');
1 row(s) returned
"GG0mFsEXb-02_dzFPqRV1Q"	toqeczld@gmail.com	CKt7l6Wq
//expected output: "GG0mFsEXb-02_dzFPqRV1Q"	toqeczld@gmail.com	CKt7l6Wq

// CALL GetUserByUid('"not exist"');
0 row(s) returned
//expected output: 0 row(s) returned